variables.mk

```diff
@@ -174,4 +162,3 @@ sim_vsrcs = \
 # assembly/benchmark variables
 #########################################################################################
 timeout_cycles = 10000000
-bmark_timeout_cycles = 100000000
```

Comment:
This is helpful for people who have benchmarks that take longer than normal tests, but still want normal tests to timeout quickly.

Reply:
I didn't see this get used... maybe I missed something but Ill add it back in.

Reply:
...another feature lost. Will make a PR this summer to fix this.

Reply:
I will not remove this then. Ill add it back